libc/docs/dev/header_generation.rst

```diff
@@ -1,120 +1,160 @@
+.. role:: raw-html(raw)
```

Comment:
is this needed?

Reply:
It is required to force line break in step 5 of Common Errors. I can't use line block syntax because there is a vertical line in the text, and escape character did not work there.